.github/workflows/release-nightly-ota.yml

```diff
@@ -247,7 +247,7 @@ jobs:
     strategy:
       matrix:
         target: [x64]
-    runs-on: ubuntu-latest
```

Comment:
I believe we should leave this unchanged, as it's only for release, not for building.